src/components/Editor/Footer.js

```diff
@@ -155,7 +155,7 @@ class SourceFooter extends PureComponent<Props> {
 
   renderSourceSummary() {
     const { mappedSource, jumpToMappedLocation, selectedSource } = this.props;
-    if (mappedSource) {
+    if (mappedSource && mappedSource != selectedSource) {
```

Comment:
what's this doing?

Reply:
It's preventing the "(from source.js)" message from displaying when you're on source.js.  Recently `mappedSource` has been introduce for all sources.  Still digging to find out how/when.

Reply:
perhaps we should move this into its own PR so we can look into it separately and land it separately?